spacepy/pybats/rim.py

```diff
@@ -289,14 +276,14 @@ def calc_I(self):
         Iup:   The total upward current over one hemisphere.
         Idown: The total downward current over one hemisphere.
 
-        Values are stored in the object with the prefix 'n_' or 's_'
+        Values are stored in the object with the prefix 'n' or 's'
         to indicate the hemisphere and may be accessed via self['n_I'], etc.
 
         Values are stored in units of mega Amps.
 
         Parameters
         ==========
-
+        N/A
```

Comment:
Just leave the section out if it's not used.

Reply:
Resolved